string pcd_path
sensor_msgs/PointCloud2 point_cloud
---
pictobot_perception/PathTraceArray pathtrace_array
geometry_msgs/PoseArray pose_array